src/plugins/console/server/lib/spec_definitions/json/generated/cluster.stats.json

```diff
@@ -11,6 +11,6 @@
       "_cluster/stats",
       "_cluster/stats/nodes/{nodes}"
     ],
-    "documentation": "https://www.opensearch.org/guide/en/elasticsearch/reference/master/cluster-stats.html"
+    "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/master/cluster-stats.html"
```

Comment:
is this customer facing?

Reply:
Do you mean this particular url or the json files within the `generated` folder?

Reply:
Shall we change all of them to `opensearch.org` so that we have consistent url pattern across the code base. For some of the broken links, we can add keep it `lycheeexculde` allowlist to avoid false alarm and add it to `noDocument `list for reference.

Reply:
Yes I think all of the customer facing links should go to the OpenSearch docs. If the particular page doesn't exist we should link to the home page for OpenSearch docs and it should be documented in the `noDocumentation` list.